src/System.Private.CoreLib/src/System/Environment.cs

```diff
@@ -76,7 +76,11 @@ public static int CurrentManagedThreadId
         {
             get
             {
+#if PLATFORM_UNIX
```

Comment:
(These are our guidelines for cross-plat code https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/cross-platform-guidelines.md)